apps/image_augmentation/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ find_package(OpenCV QUIET)
 find_package(AMDRPP QUIET)
 
 # Application Includes & Libraries
-include_directories(${ROCM_PATH}/${CMAKE_INSTALL_INCLUDEDIR}/mivisionx)
+include_directories(${ROCM_PATH}/include/mivisionx/rocal)
```

Comment:
@shobana-mcw This cannot be hardcoded. rocm team needs it for spack and other packages

Reply:
@shobana-mcw Yes Shobana, you can do it in this PR if it's just those one line changes. Please test the changes